src/answer.js

```diff
@@ -25,4 +25,6 @@ add('kokowei', require('./koko-builder.js').getKoko);
 add(['dattel', 'sultan', 'großwesir'], require('./koko-builder.js').getDatepalm);
 add(['a5', 'rhein', 'main', 'breakdance', 'club', 'feiern', '11:30'], require('./chris.js').safe);
 add(['safe'], require('./chris.js').inverseSafe);
+add(['remind'], require('./chris.js').remind);
```

Comment:
We should move all imports to the top to avoid repetition. Also remember to use `const` for those